.github/ISSUE_TEMPLATE/issue-report.md

```diff
@@ -0,0 +1,26 @@
+---
+name: Issue report
+about: Create an issue to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Describe the issue**
+A clear and concise description of what the issue is.
+
+**Environment (please complete the following information):**
+ - Host OS and version: [e.g. Oracle Linux 8.2 UEK6, macOS 10.14.6, Windows 10 Pro, ...]
```

Comment:
I would separate host OS from kernel version, i.e. have another question, "If you're using Linux, please provide the output from `uname -a`.